app/design/frontend/Magento/blank/web/css/source/_buttons.less

```
@@ -25,7 +25,8 @@
         .lib-link-as-button();
     }
 
-    .action.primary {
+    .action.primary,
+    .action-primary  {
```

Hello, thank you for your contribution! Can you please address those 2 issues:
1. Static test is failing because of double space:
```suggestion
    .action-primary {
```
2. Can you please make the same change for Luma theme's `_buttons.less`?